### ISSUE TITLE
Unexpected Error after deleting design

### ISSUE THREAD

**Issue by sokos** [avocode/avocode#1969]
Deleting last design in a Page
![screen shot 2018-04-09 at 17 21 21](https://user-images.githubusercontent.com/470663/38500093-77dafc66-3c1a-11e8-9838-bbcbb2e9fe9a.png)
 causes this error. 

---
* **Version**: 2.22.1
* **Sketch plugin version**: 49.0.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 50887
---
```
TypeError: Cannot read property 'revisions' of undefined
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:47152)
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:50544)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1773144
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4364:24
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.every (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4363:12)
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1773103)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1489392)
    at t.value (file:///Applications
```

**Comment by vojtatranta** [comment]
This will be fixed in the next release.

**Comment by jakubzitny** [comment]
This should be fixed in latest version 2.23.0. Let us know if you have any more problems :wink:
